### PR TITLE
Setup bugfixes

### PR DESCRIPTION
**Pull request by tarkatronic**
This fixes two small issues I ran into when setting up a local copy of this.

1. The `lowercaser` import in `peopleSearch` was causing troubles since it had the file extension included
2. An error message confused me for a good while since it was calling out the wrong configuration variable.

The rest of the changes in here are just beautification done by VSCode. 😄 

I am sure I will have more updates later as I adapt this to work for our purposes, but I wanted to introduce small batches to make them more digestible!

### CODE REVIEW

**Comment by tarkatronic** [comment]
@jeffwilcox Is there a CCLA to be signed by our company, or only the individual CLA?

**Comment by jeffwilcox** [comment]
@tarkatronic if possible to sign the individual CLA on behalf of the company, that is preferred, or in some cases, a contribution from a corporate account may be possible (https://github.com/cla-assistant/cla-assistant/pull/462).

Our company does not support corporate CLAs in the broad sense at scale.

**Comment by tarkatronic** [comment]
Hey @jeffwilcox just a quick update; I'm waiting for approval from legal on the CLA.

In the meantime, continuing another branch of work to expand the Passport functionality a bit for our needs, and hopefully make it a bit more generic. As well as leaning in more heavily to TypeScript as I touch bits of code!

**Comment by tarkatronic** [comment]
@jeffwilcox CLA done. Ready for contributions!

**Comment by jeffwilcox** [comment]
Awesome, thank you.

Let me start taking a look at this, and, we may need to find a good way to communicate through some of these 1:1, too, if it makes sense. My email is jwilcox at microsoft.

Jeff

**Comment by jeffwilcox** [comment]
On the Passport - super interested in that refactoring, the original design painfully bleeds through in altering the experience.

File: routes/peopleSearch.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { RequestWithSystemwidePermissions } from '../transitional';
 
 import { ensureAllLinksInMemory, getAllLinksFromRequest } from '../middleware/business/allLinks';
 
-const lowercaser = require('../middleware/lowercaser.js');
+const lowercaser = require('../middleware/lowercaser');
```
Comment (jeffwilcox) [inline routes/peopleSearch.ts:16]: General note as we TypeScript... these should become imports once the file exports the method instead... using require is buggy these days in that it hides runtime errors of a small class.

**Pull request merged**